packages/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -267,6 +270,34 @@ private void updateSelectionIfNeeded(ReactAztecText view, @Nullable ReadableMap
         }
     }
 
+    private boolean isHeadingBlock(ReactAztecText view) {
+        String tag = view.getTagName();
+        final String regex = "h([1-6])";
+        final Pattern pattern = Pattern.compile(regex);
+        final Matcher matcher = pattern.matcher(tag);
+
+        return matcher.find();
+    }
+
+    private float getHeadingScale(String scale) {
```

Comment:
This is more of a question than a change request. What is the source of this numbers and are they used in other parts of the code?

Reply:
Sure! These values were taken from [here](https://github.com/wordpress-mobile/AztecEditor-Android/blob/trunk/aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecHeadingSpan.kt#L94-L100). I tried to move them to the [Constants](https://github.com/wordpress-mobile/AztecEditor-Android/blob/trunk/aztec/src/main/kotlin/org/wordpress/aztec/Constants.kt) file using something like:

```kotlin
object Constants {
...
    val SCALE_H1 = 1.73f
    val SCALE_H2 = 1.32f
    val SCALE_H3 = 1.02f
    val SCALE_H4 = 0.87f
    val SCALE_H5 = 0.72f
    val SCALE_H6 = 0.60f
}
```

But to get them on `ReactAztecManager` I only found a way to get them using `Constants.INSTANCE.getSCALE_H1();` and apparently, it's not good for performance 🤔 

Maybe it'd be a good idea to reference where those values are from within `getHeadingScale`.

Reply:
> to reference where those values are from within `getHeadingScale`

I think that's a good idea for future reference 👍